app/tutorials/highway-management/src/main/java/org/eclipse/mosaic/app/tutorial/communication/RoundTripMessage.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2020 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.app.tutorial.communication;
+
+import org.eclipse.mosaic.lib.objects.v2x.EncodedPayload;
+import org.eclipse.mosaic.lib.objects.v2x.MessageRouting;
+import org.eclipse.mosaic.lib.objects.v2x.V2xMessage;
+
+import java.io.ByteArrayOutputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import javax.annotation.Nonnull;
+
+public class RoundTripMessage extends V2xMessage {
+
+    /**
+     * The encoded message.
+     */
+    private final EncodedPayload encodedPayload;
+
+    /**
+     * Simple message taking a byte array as input for the payload.
+     *
+     * @param routing the {@link MessageRouting} for the message
+     * @param payload the byte array
+     */
+    public RoundTripMessage(MessageRouting routing, final byte[] payload) {
+        super(routing);
+        try (final ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
+             final DataOutputStream dataOutputStream = new DataOutputStream(byteArrayOutputStream)) {
+            dataOutputStream.write(payload);
+            this.encodedPayload = new EncodedPayload(byteArrayOutputStream.toByteArray(), byteArrayOutputStream.size());
```

Comment:
Sorry, I'm picky today 😬 

Why not just 
```
this.encodedPayload = new EncodedPayload(payload, payload.length);
```
As this is just a test message. You also do not really need a payload, you can also just create an `EncodedPayload` object with a `length` parameter only. 

```
this.encodedPayload = new EncodedPayload(8);
```

Reply:
Good suggestion. This was pretty much copy-pasted from a different `V2xMessage`